src/timeMachine/components/view_options/LegendOrientation.tsx

```diff
@@ -45,6 +49,32 @@ interface OwnProps {
 type ReduxProps = ConnectedProps<typeof connector>
 type Props = OwnProps & ReduxProps
 
+const HORIZONTAL = 'horizontal'
+const VERTICAL = 'vertical'
+
+/**
+ *  The LegendOrientation Component consists of three properties:
+ *  1.  The Orientation itself (horizontal vs. vertical); horizontal is the default
+ *  2.  opacity of the legend (ranges from 20% to 100%); default is 100%
+ *  3.  row colorization:  does each row have colored text (the default), or does each row have a dot of color prefacing it, with
+ *  text all the same default (foreground) color
+ *
+ *  The above is what the user sees.
+ *
+ *  what is actually happening:
+ *
+ *    1.  Legend Orientation.  Default is horizontal
+ *          behind the scenes it is a 'rotation threshold'.  if the number of items is above the threshold, then
+ *          the legend is displayed vertically (each line is a column); if the number of items is below the threshold, then the legend is
+ *          displayed horizontally (each line is a row).  therefore, since the default is horizontal, the numbers being set are very high
+ *          for horizontal (high enough that the database would thrash out/throttle before it got that high), or low (0 for vertical, since
+ *          to display something it has to be higher than zero).  The default number itself has changed, so some users may see their graphs flip
+ *          (before; without the feature flag enabled, the default was hard-coded to 10; so graphs with more than 10 lines of  data in the tooltip
+ *          were vertical)
+ *
+ *    2.  Legend Opacity.  The actual number is between the minimum .2 and the maximum 1.0; but we are displaying it as a matter of percentage
+ *    3.  Row Colorization.  This is the simplest.  just a true/false flag, it is true by default.
+ */
```

Comment:
i love the legend orientation explanation, bu i think this a little too dense for an in code comment. i think it's okay to ✂️ this

Reply:
i was asked specifically to put an explanation like this in the code since we are really just using the threshold, not a flag.  if it's not here, where should it go?

Reply:
if you were asked to put it here, then great work! ignore my previous comment :)